Example/BugShaker/AppDelegate.swift

```diff
@@ -20,7 +20,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     *  Configure BugShaker with an array of email recipients (required)
     *  and an optional custom subject line to use for all bug reports.
     */
-    BugShaker.configure(to: ["example@email.com"], subject: "Bug Report")
+
+    let iOSVersion = UIDevice.currentDevice().systemVersion
+    BugShaker.configure(to: ["example@email.com"], subject: "Bug Report", body: "Hi Developers, \n I am running your app on \(iOSVersion ?? "unknown") and wanted to thank you.\n I am also reporting a bug where ... ")
```

Comment:
I think just a short & simple text intro for the body would work for the example. The iOS version seems to go a bit beyond scope of this PR. Also, the lengthy body text in the example makes the example a little harder to read.

I will be adding device info (i.e. OS version) as a standard component in these reports, very soon.